src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -102,6 +103,8 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	private static readonly MOUSE_WHEEL_EVENT_THRESHOLD = 150;
 	private static readonly MOUSE_WHEEL_DISTANCE_THRESHOLD = 1.5;
 
+	private editorPinnedAndUnpinnedTabsContext: IContextKey<boolean>;
```

Comment:
I have another idea: we could make this a context that tracks if a group has pinned tabs and move this into the existing handler that already tracks pinning state: https://github.com/microsoft/vscode/blob/e00d61b438c9636f49a9a50df0171a2d7d3cb134/src/vs/workbench/browser/parts/editor/editorGroupView.ts#L236